flink-streaming-java/src/main/java/org/apache/flink/streaming/util/latency/LatencyHistogramStatistics.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.streaming.util.latency;
+
+import org.apache.flink.metrics.HistogramStatistics;
+
+import org.apache.commons.math3.stat.descriptive.DescriptiveStatistics;
+
+
+/**
+ * Latency histogram statistics implementation returned by {@link LatencyHistogram}.
+ * The statistics class wraps a {@link DescriptiveStatistics} instance and forwards the method calls accordingly.
+ */
+public class LatencyHistogramStatistics extends HistogramStatistics {
+	private final DescriptiveStatistics latencyHistogram;
+
+	public LatencyHistogramStatistics(DescriptiveStatistics latencyHistogram) {
+		this.latencyHistogram = latencyHistogram;
+	}
+
+	@Override
+	public double getQuantile(double quantile) {
+		return latencyHistogram.getPercentile(quantile);
```

Comment:
This is wrong. The quantile must be multiplied by 100, since the range of acceptable values for Histogram#getQuantile is 0-1, whereas for DescriptiveStatistics it is 1-100.

Reply:
sorry, I will fix.